superbench/tools/system_info.py

```diff
@@ -421,15 +430,21 @@ def get_network(self):
             dict: dict of network info.
         """
         network_dict = {}
-        network_dict['nic'] = self.__get_nic()
-        network_dict['ib'] = self.__get_ib()
-        ofed_version = self.__run_cmd('ofed_info  -s').strip()
-        network_dict['ofed_version'] = ofed_version
+        try:
+            network_dict['nic'] = self.get_nic()
+            network_dict['ib'] = self.get_ib()
+            ofed_version = self._run_cmd('ofed_info  -s').strip()
+            network_dict['ofed_version'] = ofed_version
+        except Exception as ex:
+            logger.exception('Error: get network info failed, message: {}'.format(str(ex)))
         return network_dict
 
     def get_all(self):
         """Get all system info and save them to file in json format."""
         sum_dict = {}
+        if os.geteuid() != 0:
+            logger.error('You need to be as a root user to run this tool.')
+            return sum_dict
```

Comment:
pls update the unit test accordingly
https://github.com/microsoft/superbenchmark/blob/890ce65daeb43ecac2dced0eb6c7fd1d9d5b17dd/tests/cli/test_sb.py#L87
"expect_failure=True" -> "expect_failure=False"